pylutron/__init__.py

```diff
@@ -749,29 +783,155 @@ def type(self):
   @property
   def is_dimmable(self):
     """Returns a boolean of whether or not the output is dimmable."""
-    return self.type != 'NON_DIM' and not self.type.startswith('CCO_')
+    return self.type not in (
+      'NON_DIM', 'NON_DIM_INC', 'NON_DIM_ELV', 'EXHAUST_FAN_TYPE',
+      'RELAY_LIGHTING') and not self.type.startswith(
+      'CCO_')
 
+  @property
+  def is_motor(self):
+    """True if this is a motor kind of output, like shade, motor etc - that can go up and down"""
+    return self.type in ('MOTOR', 'SYSTEM_SHADE')
 
-class Shade(Output):
```

Comment:
After seeing your question I figured that since I only have an HW install at home, I missed the fact that RadioRA (which is a much simpler version) has no support for #SHADEGRP (and it only uses the OUTPUT to control shades).
As such, do you think its better to make a whole new (very similar) library for HW, or do the logic in the existing library.

Reply:
sorry, how can I send you that file, its rather large....

Reply:
In any case, I will drop this push, and will redo as you suggested.
Don't you think its better to break the file into multiple files as its rather long (i.e. by class for example).